docs/introduction/Cross-chain-transfer-protocol.rst

```diff
@@ -41,6 +61,41 @@ A backward transfer moves coins back from a sidechain to the mainchain destinati
 A Backward Transfer is initiated by a **Withdrawal Request** which is a sidechain transaction issued by the coin's owner. The request specifies the mainchain destination address and the amount. More precisely, the withdrawal request owner will create a WithdrawalRequestBox that destroys the specified amount of coins in the sidechain. This is not enough to move those coins back to the mainchain though: we need to wait until the end of the withdrawal epoch, when all the coins specified in that epoch’s Withdrawal Requests are listed in a single certificate, that is then propagated to the mainchain.
 The certificate includes a succinct cryptographic proof that the rules associated with the declared verifying key have been respected. Certificates are processed by the mainchain consensus, which recreates the coins as specified by the certificate, only checking that the proof verifies, and that the coins received by a sidechain match the amount that was sent to it.
 
+Can be performed by following RPC command on the Mainchain side:
+
+.. code:: Bash
+
+    sc_request_transfer(outputs, params)
+
+Where outputs is array of json objects representing the amounts to send. Each array element must contain following elements
+   scid           - sidechain ID in uint256 format
+   vScRequestData - sidechain Utxo ID for which a backward transafer is being requested
+   pubkeyhash     - mainchain address where to send the backward transferred amount
+   scFee          - value spent by the sender that will be gained by a SC forger
+
+Params is a json object with the following command parameters
+   fromaddress   - The address to send the funds from. If omitted funds are taken from all available UTXO.
+   changeaddress - The address to send the change to, if any. If not set, fromaddress is used. If the latter is not set too, a new generated address will be used.
+   minconf       - Minimum times of confirmation the funds should have.
+   fee           - The fee amount to attach to this transaction.
+
+Ceased Sidechain Withdrawal
+===========================
+
+When sidechain is ceased funds from sidechain can be retained to the mainchain with Ceased Sidechain Withdrawal request. This request can be performed after 2 withdrawal epochs after sidechain ceasing.
+This request can be performed by raw transaction with following structure
```

Comment:
This request can be performed through a raw transaction with the following structure